### DISCUSSION TITLE
Connected SSID name?

### DISCUSSION THREAD

**Discussion by navdeepssidhu** [arendst/Tasmota#15882] [Q&A] [answered]
Hi,
How can i know the connected  ssid (ssid1 or ssid2) ? 
Is it possible to make rule for example:  if ssid1 connected, then power1 on and if ssid2 connected power2 on ?
any help is appreciable.
Thanks

**Comment by sfromis** [comment, marked as answer]
The regular `.../STATE` payloads published includes Wifi details, which are available to rules, like this:
`"Wifi":{"AP":1,"SSId":"mynetwork","BSSId":"router-ip-address","Channel":1,"Mode":"11n","RSSI":100,"Signal":-41,"LinkCount":1,"Downtime":"0T00:00:08"}`

**Reply by navdeepssidhu** [reply]
Thanks for the reply, I am just a beginner using Tasmota as On Off device. I tried the below rule to find AP name or AP number (1 or 2) but not getting the proper value. Could you please guide. Thanks
 
Rule1 ON wifi#connected DO Var1 %APNAME% ENDON

**Reply by sfromis** [reply]
Do not invent stuff not in the docs by randomly guessing. You have no more % variables than: https://tasmota.github.io/docs/Rules/#rule-variables.

Using the JSON you see in the messages, you can create JSON-based rule triggers to pick up values as they are published, which is **not** at `Mqtt#Connected` time. https://tasmota.github.io/docs/Rules/#rule-trigger
